src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\Xml\Core\XmlSubtreeReader.cs" />
     <Compile Include="System\Xml\Core\XmlSubtreeReaderAsync.cs" />
     <Compile Include="System\Xml\Core\XmlTextEncoder.cs" />
-    <Compile Include="System\Xml\Core\BinaryCompatibility.cs" />
```

Comment:
@weshaggard Doesn't it need a reference assembly for it? 
`System/Xml/Core/XmlTextReaderImplAsync.cs(3363,17): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)`